src/Validation/Validator.php

```diff
@@ -526,6 +582,19 @@ public function allowEmpty($field, $when = true)
      * $validator->notEmpty('email'); // Email cannot be empty
      * $validator->notEmpty('email', $message, 'create'); // Email can be empty on update
      * $validator->notEmpty('email', $message, 'update'); // Email can be empty on create
+     * $validator->notEmpty(['email', 'title'], $message, 'update');
+     * // Email and title can be empty on create but can not on update
+     * $validator->notEmpty(
+     *      [
+     *          'email',
+     *          'title' => [
+     *              'when' => true,
```

Comment:
this is how its been in code (https://github.com/cakephp/cakephp/blob/3.next/src/Validation/Validator.php#L509) i also dont know why its not 'on' but 'when' so ive left just left it same, i can change it for consistency if you want